docs/getting_started/basic_usage/basic_usage.rst

```diff
@@ -115,6 +83,27 @@ network
     e.g. you can choose between different LSTM implementations, or GRU, or standard RNN, etc.
     See :class:`TFNetworkRecLayer.RecLayer` or :class:`NetworkRecurrentLayer.RecurrentUnitLayer`.
     See also :ref:`tf_lstm_benchmark`.
+..
+    batching
+        The sorting variant when the mini-batches are created. E.g. ``random``.
```

Comment:
Either put it back in, or completely remove.
I don't think that we should have commented out text in the documentation files.
If you really want to remove that, the `seq_ordering` option for the datasets should be emphasized more (which is completely missing currently).

Reply:
The `seq_ordering` option is mentioned in the full parameter overview (in the entry for `batching`). This is also the reason why I removed `chunking` here, the same description is available under configuration reference / training. I also have a sentence recommending `seq_ordering` over `batching`, but this is not part of this PR yet.

Reply:
Maybe to not cause confusion: at this location (basic usage), I only wanted to state the most general parameters, which are needed for every model. Everything else is sorted into the parameter categories for better visibility.

Reply:
But I would argue that it is pretty basic to have sequence shuffling.
But anyway, there should not be commented out text.

Reply:
Ok. If your are okay with it i would leave batching in (with the seq_ordering note), and chunking out, as there is a link to the other parameters below.